index.html

```diff
@@ -23,13 +23,254 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
             don't hestiate to contact me.</p>
             
         <div class="social-media">
-            <img src="./Microverse-Student-Project/Tertiary/Disabled.png" alt="Github icon">
-            <img src="./Microverse-Student-Project-1(1)/Tertiary/Disabled.png" alt="Linkedin icon">
-            <img src="./Microverse-Student-Project-1(2)/Tertiary/Enabled.png" alt="icon">
-            <img src="./Microverse-Student-Project-1(3)/Tertiary/Disabled.png" alt="Twitter icon">
-            <img src="./Microverse-Student-Project-1(4)/Tertiary/Disabled.png" alt="Medium icon">   
+            <img src="./images/Microverse-Student-Project/Tertiary/Disabled.png" alt="Github icon">
+            <img src="./images/Microverse-Student-Project-1(1)/Tertiary/Disabled.png" alt="Linkedin icon">
+            <img src="./images/Microverse-Student-Project-1(2)/Tertiary/Enabled.png" alt="icon">
+            <img src="./images/Microverse-Student-Project-1(3)/Tertiary/Disabled.png" alt="Twitter icon">
+            <img src="./images/Microverse-Student-Project-1(4)/Tertiary/Disabled.png" alt="Medium icon">   
         </div>    
     </section>
+                <!--Works Section-->
+
+    <div class="grid-container">
+        <header class="title">
+            <h1 class="title-h1">My Recent Works</h1>
+            <hr class="title-hr">
+            <div class="frame">
+                <div class="featured">
+                   <div class="right-block">
+                        <img class="feature-img" src="./images/ImgPlaceholder.png" alt="">
+                        <h2 class="title-post">Multi-Post Stories</h2>
+                        <p class="feature-supporting-text">
+                            A daily selection of privately personalized
+                            reads; no accounts or sign-ups required.
+                            has been the industry's standard dummy
+                            text ever since the 1500s, when an
+                            unknown printer took a standard dummy text.
+                        </p>
+                        <div class="tags">
+                            <ul class="tag1">
+                                <li>css</li>
+                            </ul>
+                            <ul class="tag2">
+                                <li>html</li>
+                            </ul>
+                            <ul class="tag3">
+                                <li>bootstrap</li>
+                            </ul>
+                            <ul class="tag4">
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+                        <button type="submit" class="button">See Project</button>
```

Comment:
- [ ] [OPTIONAL] Please change the button interaction styling of the `See Project` button to look like what's in the Figma design. 

- [ ] [OPTIONAL] There is no bottom margin for the `See Project` button in the design. So please make sure it has got no bottom margin spacing. 

![Buttons](https://user-images.githubusercontent.com/12524453/176964434-2b2c48c3-f970-42c2-93ea-35421fb21282.png)